### ISSUE TITLE
Install to USB stick then boot that from a PC

### ISSUE THREAD

**Issue by biojet1** [meefik/linuxdeploy#893]
I want to install any x86_64 linux distribution to a usb (from micro usb port OTG) then boot that usb to a PC.
Is that possible.
My phone is Samsung J2 MT6737T ARMv7 rev 3 (v7l).

**Comment by notorand-it** [comment]
Nope. This is a containers based solution. You need a Linux host system.
This means that the installed linux will run using the very same kernel as the host system.
You can "simply burn" an x86_64 ISO image onto a USB stick. Then boot from there.

**Comment by notorand-it** [comment]
Not really based on containers. But surely it needs a working Linux host.
